scripts/asm-installer/release_asm_installer

```diff
@@ -18,7 +18,14 @@ BUCKET_PATH="csm-artifacts/asm"; readonly BUCKET_PATH
 
 CURRENT_RELEASE="release-1.9-asm"; readonly CURRENT_RELEASE
 
+ALWAYS_PUBLISH_VERSION="${ALWAYS_PUBLISH_VERSION:=0}"
+STABLE_VERSION_FILE="STABLE_VERSIONS"; readonly STABLE_VERSION_FILE
+VERSION_FILE_PATH="${BUCKET_PATH}/${STABLE_VERSION_FILE}"
+HOLD_TYPE="temp"
+trap 'gsutil retention "${HOLD_TYPE}" release gs://"${VERSION_FILE_PATH}"' ERR  # make sure the hold is cleared
```

Comment:
Do I need EXIT and SIGINT here? I tried but the command seems to run multiple times if I ctrl-c. Seems like the shell received multiple signal in addition to SIGINT.

Reply:
Almost certainly don't want EXIT or else it would run even if it exits successfully. I think ERR includes SIGINT.